src/styles/partials/components/_alert.scss

```diff
@@ -104,8 +121,22 @@
       background: $error-color;
       outline-color: $error-color;
 
+      .dg_alert__status {
+        color: $black;
+      }
+
+      .dg_alert__icon {
+        color: $black;
+      }
+      * {
```

Comment:
think you can just use * here

Reply:
Did a sweep and cleaned up properties a bit. Take a look and let me know if this looks better. This should address your comment above as well. @martypowell